### PR TITLE
Uplift k8s to 1.20.4

### PR DESCRIPTION
**Pull request by kashifest**
Since kind node image version is not a 1 to 1 mapping to k8s version we also introduce `KIND_NODE_IMAGE_VERSION` variable.

### CODE REVIEW

**Comment by kashifest** [comment]
/hold 
until node images are built

**Review by fmuyassarov — COMMENTED** [review]
Maybe https://github.com/kubernetes/kubernetes/releases/tag/v1.20.5 ?

**Comment by kashifest** [comment]
> Maybe https://github.com/kubernetes/kubernetes/releases/tag/v1.20.5 ?

Nope, we always keep a gap to allow upgrade

**Review by fmuyassarov — APPROVED** [review]

File: lib/common.sh [hunk 1]
```diff
@@ -199,6 +199,7 @@ export KUSTOMIZE_VERSION=${KUSTOMIZE_VERSION:-"v3.8.5"}
 
 # Kind version
 export KIND_VERSION=${KIND_VERSION:-"v0.10.0"}
+export KIND_NODE_IMAGE_VERSION=${KIND_NODE_IMAGE_VERSION:-"1.20.2"}
```
Comment (furkatgofurov7) [inline lib/common.sh:202]: I have two questions:

1. why kind image version is not 1 to 1 map to k8s version? 
2. why do we have different versions of kind image in this patch and in [airship-dev-tools repo](https://github.com/Nordix/airship-dev-tools/pull/271/files#diff-54b22a5b934099e29518bd94112e4f547eaddc3d8002c8510e59bd4188bc38b8R13)

Comment (kashifest) [reply lib/common.sh]: 1.  Probably kind folks are the best person to answer, why did they stop building kind images after 1.20.2 
2. We do not have different kind version in airship-dev-tools repo. Check the latest master.

Comment (furkatgofurov7) [reply lib/common.sh]: >     2\. We do not have different kind version in airship-dev-tools repo. Check the latest master.

Missed this [patch](https://github.com/Nordix/airship-dev-tools/pull/272) in between.

**Review by furkatgofurov7 — COMMENTED** [review]
/lgtm

**Pull request merged**